src/hooks/useHideBullet.ts

```diff
@@ -36,6 +36,8 @@ const useHideBullet = ({
     const thought = getThoughtById(state, thoughtId)
     // bullet may be set from =children or =grandchildren and passed as a prop
     if (hideBulletProp) return true
+    // and if the thought is somehow not found, we just pass the `undefinedness` up the chain
+    if (!thought) return
```

Comment:
Let's return false to preserve the boolean return type.

Reply:
I had a rationale, but keeping the type signature makes sense to me too 🙇